issue found on: analyser/aggregator.go:203:2: rangeValCopy

an issue has been detected in source code: ./analyser/aggregator.go:203:2: rangeValCopy: each iteration copies 144 bytes (consider pointers or indexing)

--- a/analyser/aggregator.go
+++ b/analyser/aggregator.go
@@ -199,8 +199,8 @@ func printReadEntries(colorizer aurora.Aurora, entries, notRead []AggregatorLogE
 }
 
 func messageWithOffsetIn(entries []AggregatorLogEntry, offset int) bool {
-	for _, entry := range entries {
-		if entry.Offset == offset {
+	for i := range entries {
+		if entries[i].Offset == offset {
 			return true
 		}
 	}

